doc/api/http.md

```diff
@@ -313,8 +313,10 @@ the data is read it will consume memory that can eventually lead to a
 Node.js does not check whether Content-Length and the length of the
 body which has been transmitted are equal or not.
 
-The request implements the [Writable Stream][] interface. This is an
-[`EventEmitter`][] with the following events:
+The request inherits from [Stream][], with two methods like what we have in
```

Comment:
Can we just omit the two methods and the mention of `Writable Stream`? The methods are documented below and mentioning `Writable Stream` here seems to confuse more than enlighten.

```markdown
The request inherits from [Stream][].
```

That's all that needs to be said. (Or is there a good reason to mention the other stuff?)

Reply:
Consider we still have events' introductions below it, so what about saying this?

>The request inherits from [Stream][], with some events, properties and methods below: